### PR TITLE
Update base1ns with new resource quotas from test tier.

### PR DESCRIPTION
**Pull request by cfchase**
Paired with https://github.com/codeready-toolchain/toolchain-e2e/pull/612

### CODE REVIEW

File: deploy/templates/nstemplatetiers/base1ns/ns_dev.yaml [hunk 1]
```diff
@@ -14,65 +14,59 @@ objects:
...
 
-# Role and RoleBindings for CRT administration (not associated with users)
-- apiVersion: rbac.authorization.k8s.io/v1
-  kind: Role
+  # Quotas and default limits for not-terminating containers (regular long-running containers)
+  # and terminating (short-lived containers like build) containers
+- apiVersion: v1
+  kind: ResourceQuota
   metadata:
-    name: exec-pods
```
Comment (cfchase) [inline deploy/templates/nstemplatetiers/base1ns/ns_dev.yaml:21]: not sure if the exec-pods / crtadmin stuff was necessary for the base1ns tier.  I need to try it out in the test tier, if that's the case.  Let me know

File: deploy/templates/nstemplatetiers/base1ns/cluster.yaml [hunk 2]
```diff
@@ -3,25 +3,6 @@ kind: Template
...
 objects:
-- apiVersion: quota.openshift.io/v1
-  kind: ClusterResourceQuota
-  metadata:
-    name: for-${USERNAME}-compute
-  spec:
-    quota:
-      hard:
-        limits.cpu: 20000m
-        limits.memory: ${MEMORY_LIMIT}
```
Comment (rajivnathan) [inline deploy/templates/nstemplatetiers/base1ns/cluster.yaml:14]: I think you removed the `MEMORY_LIMIT` below? You'd have to either add it back or fill in the value here instead.

File: deploy/templates/nstemplatetiers/base1ns/cluster.yaml [hunk 3]
```diff
@@ -3,25 +3,6 @@ kind: Template
...
-        limits.ephemeral-storage: 7Gi
-        requests.cpu: 1750m
-        requests.memory: ${MEMORY_REQUEST}
```
Comment (rajivnathan) [inline deploy/templates/nstemplatetiers/base1ns/cluster.yaml:17]: Same here

File: deploy/templates/nstemplatetiers/base1ns/ns_dev.yaml (hunk 1 above)
Comment (rajivnathan) [reply deploy/templates/nstemplatetiers/base1ns/ns_dev.yaml:21]: I think it is required for administration purposes like the comment above suggests so I think we should restore it.

**Review by rajivnathan — CHANGES_REQUESTED** [review]

File: deploy/templates/nstemplatetiers/base1ns/cluster.yaml (hunk 2 above)
Comment (cfchase) [reply deploy/templates/nstemplatetiers/base1ns/cluster.yaml:14]: This clusterresourcequota was deleted, hence MEMORY_LIMIT and MEMORY_REQUEST were deleted below.

Comment (rajivnathan) [reply deploy/templates/nstemplatetiers/base1ns/cluster.yaml:14]: Not sure why I thought this was an add and not a remove 🤦 My bad...

File: deploy/templates/nstemplatetiers/base1ns/ns_dev.yaml (hunk 1 above)
Comment (cfchase) [reply deploy/templates/nstemplatetiers/base1ns/ns_dev.yaml]: done

**Review by rajivnathan — APPROVED** [review]

**Comment by alexeykazakov** [comment]
/retest

(Fixed e2e tests)

**Comment by alexeykazakov** [comment]
/retest

(Fixed e2e tests)

**Comment by alexeykazakov** [comment]
/retest

OpenShift CI infra issue

**Pull request merged**